t/cli/test_main.sh

```diff
@@ -310,6 +310,35 @@ fi
 
 echo "pass: support environment variables in local_conf"
 
+# support default value when environment not set
+echo '
+etcd:
+    host:
+        - "http://${{ETCD_HOST:192.168.1.1}}:${{ETCD_PORT}}"
+' > conf/config.yaml
+
+ETCD_PORT=2379 make init
+
+if ! grep "env ETCD_HOST=192.168.1.1;" conf/nginx.conf > /dev/null; then
```

Comment:
I think it would be more appropriate to check if etcd is connected properly from the APISIX error.log.

Reply:
When etcd cannot connect correctly, I found that error will print in console, not error.log